### ISSUE TITLE
Add native text-to-speech

### ISSUE THREAD

**Issue by avi12** [brave/brave-browser#26474]
## Description 

I recently migrated from Edge, and one of the features I miss the most is the built-in text-to-speech ("Read aloud") that includes natural voices.
It also provided TTS for lesser-supported languages like Hebrew.

## Brave version (brave://version info)

1.45.116

## Version/Channel Information:

- Can you reproduce this issue with the current release? **Yes**

## Other Additional Information:

- Does the issue resolve itself when disabling Brave Shields? **No**
- Does the issue resolve itself when disabling Brave Rewards? **No**
- Is the issue reproducible on the latest version of Chrome? **Yes**

## Miscellaneous Information:

[Read aloud](https://microsoftedgetips.microsoft.com/en-us/collections/productivity/12?form=MA13DX) is a feature that allows highlighting any text on the web page and starting the TTS engine from that starting point.

**Comment by nour-dev** [comment]
Can this be supported in brave android too ? Like audify app

**Comment by chris122380** [comment]
> Can this be supported in brave android too ? Like audify app

I tried enabling the Read aloud flag in Brave for Android and it wouldn't show in the menu 😢. Now that Google has decided to replace Google Assistant with Gemini and Gemini won't read aloud articles. It would be nice to be able to have Brave read aloud articles in the browser.

**Comment by I-I-IT** [comment]
Agreed, a TTS will be mega useful, esp on Android. I am not even sure this needs AI.

**Comment by ZackMercury** [comment]
No implementation in 4 years?
